indico/htdocs/js/indico/modules/timetable/timetable/Draw.js

```diff
@@ -778,7 +778,7 @@ type("TimetableBlockNormalManagement", ["TimetableBlockNormal", "TimetableBlockM
      {
          _getTitle: function(){
              var title = this.TimetableBlockNormal.prototype._getTitle.call(this);
-             return this.eventData.friendlyId ? this.eventData.friendlyId + " - " + title : title;
+             return parseInt(this.eventData.friendlyId) >= 0 ? this.eventData.friendlyId + " - " + title : title;
```

Comment:
is it ever not defined? also, don't use parseInt without the second argument (`+this.eventData.friendlyId >= 0` would probably work too)